feature/identity/src/main/java/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -24,7 +25,12 @@ class IdentityManager(
 
     @Synchronized
     fun configure(appUserID: String?) {
+        if (appUserID?.isEmpty() == true) {
```

Comment:
I imagine there's nothing weird with Kotlin here and this only evaluates to `true` if the result is not `null` and `true`.